src/snowcli/pycharm_remote_debug.py

```diff
@@ -0,0 +1,30 @@
+from typing import Optional
+
+from click import ClickException
+
+
+def setup_pycharm_remote_debugger_if_provided(
+    pycharm_debug_library_path: Optional[str],
+    pycharm_debug_server_host: Optional[str],
+    pycharm_debug_server_port: Optional[int],
+):
+    if pycharm_debug_library_path:
+        if (
+            pycharm_debug_server_host is not None
+            and pycharm_debug_server_port is not None
+        ):
+            import sys
+
+            sys.path.append(pycharm_debug_library_path)
+            import pydevd_pycharm
```

Comment:
Should this be added as dev dependcy to pyproject.toml?

Reply:
JetBrains instructions say that this library is versioned together with PyCharm IDE and that the versions have to match.

Line from their instruction:
```
pip install pydevd-pycharm~=<version of PyCharm on the local machine>
```

So I don't think it's possible to add it to pyproject.toml.

Reply:
Ok, got it. I assume this `pip install` is in the link, right?